dotnetapp-3.5/dotnetapp-3.5/Dockerfile.build

```diff
@@ -0,0 +1,16 @@
+FROM microsoft/dotnet-framework:3.5
+
+ENV MSBUILD_VERSION 14.0 
+ENV MSBUILD_DOWNLOAD_URL https://download.microsoft.com/download/E/E/D/EEDF18A8-4AED-4CE0-BEBE-70A83094FC5A/BuildTools_Full.exe 
+
+RUN powershell -NoProfile -Command \ 
+         $ErrorActionPreference = 'Stop'; \ 
+         Invoke-WebRequest %MSBUILD_DOWNLOAD_URL% -OutFile BuildTools_Full.exe; \ 
+         & BuildTools_Full.exe /Silent /Full
```

Comment:
@rakeshsinghranchi, @vivmishra  - I didn't see a response to this comment?  We talked about this in person, I thought this was going to be addressed?